tests/TestCase/ORM/TableTest.php

```diff
@@ -381,10 +381,59 @@ public function testDisplayFieldLabel(): void
         $this->assertSame('label', $table->getDisplayField());
     }
 
+    /**
+     * Tests that displayField will fallback to first *_name field
+     */
+    public function testDisplayNameFallback(): void
+    {
+        $table = new Table([
+            'table' => 'users',
+            'schema' => [
+                'id' => ['type' => 'integer'],
+                'custom_title' => ['type' => 'string'],
+                '_constraints' => ['primary' => ['type' => 'primary', 'columns' => ['id']]],
+            ],
+        ]);
+        $this->assertSame('custom_title', $table->getDisplayField());
+
+        $table = new Table([
+            'table' => 'users',
+            'schema' => [
+                'id' => ['type' => 'integer'],
+                'name' => ['type' => 'string'],
+                'custom_title' => ['type' => 'string'],
+                '_constraints' => ['primary' => ['type' => 'primary', 'columns' => ['id']]],
+            ],
+        ]);
+        $this->assertSame('name', $table->getDisplayField());
+
+        $table = new Table([
+            'table' => 'users',
+            'schema' => [
+                'id' => ['type' => 'integer'],
+                'title_id' => ['type' => 'integer'],
+                'custom_name' => ['type' => 'string'],
+                '_constraints' => ['primary' => ['type' => 'primary', 'columns' => ['id']]],
+            ],
+        ]);
+        $this->assertSame('custom_name', $table->getDisplayField());
+
+        $table = new Table([
+            'table' => 'users',
+            'schema' => [
+                'id' => ['type' => 'integer'],
+                'nullable_title' => ['type' => 'string', 'null' => true],
+                'custom_name' => ['type' => 'string'],
+                '_constraints' => ['primary' => ['type' => 'primary', 'columns' => ['id']]],
+            ],
+        ]);
+        $this->assertSame('custom_name', $table->getDisplayField());
+    }
+
     /**
      * Tests that no displayField will fallback to primary key
      */
-    public function testDisplayFallback(): void
+    public function testDisplayIdFallback(): void
```

Comment:
Why id
Isnt it field?

Reply:
No, because that test checks if it fallbacks to the primary key which is the `id` field in that specific schema
https://github.com/cakephp/cakephp/blob/a9942493c3116d078450b74160d886544ecde559/tests/TestCase/ORM/TableTest.php#L433